src/main/kotlin/org/GreetingKotlin.kt

```diff
@@ -3,4 +3,4 @@ package org
 import org.infinispan.protostream.annotations.Proto
 
 @Proto
-data class GreetingKotlin(val name: String?, val greeting: String?)
+class GreetingKotlin(@JvmField var name: String?=null, @JvmField var greeting: String?=null)
```

Comment:
I had to add these annotations in order to make protostream pass. however, one the code is generated (the schema that is used both in the client and uploaded to the server) the app does not compile. This is unrelated to Quarkus, I experience the same issue with vanilla Infinispan
See https://github.com/infinispan/infinispan-simple-tutorials/pull/319/commits

Reply:
Thanks for the update, so there is no data class support?